.circleci/config.yml

```diff
@@ -15,21 +15,18 @@ jobs:
 
       - restore_cache:
           keys:
-            - v1-dependencies-{{ checksum "package.json" }}
-            - v1-dependencies-
+            - v4-dependencies-{{ checksum "yarn.lock" }}
+            - v4-dependencies-
 
-      - run: yarn install
-
-      - save_cache:
-          paths:
-            - node_modules
-          key: v1-dependencies-{{ checksum "package.json" }}
+      - run: yarn install --frozen-lockfile
```

Comment:
It might do the trick to avoid this change to do a `yarn install --force` locally and commit the new lockfile. Also might be worth it to remove all the hats `^` from the package file to avoid "wrong" dependencies.

Sorry for being 🐌 to apply the orb.

Reply:
The reason I added `--frozen-lockfile` is that it will fail the build if `yarn.lock` and `package.json` do not correspond. 

We should remove the hats from all our normal and dev dependencies, but they should stay in the peer dependencies.